checks/contributors.go

```diff
@@ -38,6 +43,28 @@ func Contributors(c checker.Checker) checker.CheckResult {
 			if err != nil {
 				return checker.RetryResult(err)
 			}
+			// TODO - Figure out the real API for getting user orgs
+			url := fmt.Sprintf("https://api.github.com/users/%s/orgs", contrib.GetLogin())
+			resp, e := c.HttpClient.Get(url)
+
+			if e != nil {
+				c.Logf("unable to get org members for %s", contrib.GetLogin())
+			} else {
+				var orgs orgnaizations
+
+				err = json.NewDecoder(resp.Body).Decode(&orgs)
+
+				if err != nil {
+					c.Logf("unable to get  decode json for for %s org", contrib.GetLogin())
+				} else {
+					if len(orgs) > 0 {
+						companies[orgs[0].Login] = struct{}{}
+						continue
```

Comment:
Also, why continue, i think we need to check company as well.

Reply:
Your org's is not shown, because you don't have any orgs that are public.

Reply:
We shouldn't count for the same contributor in `org` and `company` and that is the reason for continue. Here is my id https://api.github.com/users/naveensrinivasan/orgs , I had to enable the public settings in every org I am in.